catalog-rest-service/src/main/resources/ui/src/pages/DashboardDetailsPage/DashboardDetailsPage.component.tsx

```diff
@@ -31,10 +34,42 @@ import { getTagCategories, getTaglist } from '../../utils/TagsUtils';
 type ChartType = {
   displayName: string;
 } & Chart;
+
+const dashboardDetailsTabs = [
```

Comment:
This should be in a constant file.
same for other entities.

Reply:
move to utils file

Reply:
Is this being used only in utils method and no where else?

Reply:
its only use in the respective page component of dataset, topic, dashboard, and pipeline